dbt_subprojects/dex/macros/to_wrapped_native_token.sql

```diff
@@ -11,6 +11,11 @@ CASE
     END AS {{to_column_name}}
 
 {% elif blockchain == 'base' %}
+CASE 
+        WHEN {{from_column_name}} = 0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee THEN 0x4200000000000000000000000000000000000006 
+        ELSE {{from_column_name}}
+    END AS {{to_column_name}}
+{% elif blockchain == 'optimism' %}
```

Comment:
logic looks the same? should be able ot:
`{% elif blockchain in ('optimism', 'base') %}`

Reply:
[adopted](https://github.com/duneanalytics/spellbook/pull/8140/commits/a81a3295d791e9762f34b8bad7311e3e37ee96eb), thank you